components/organisms/community/OptionsMenu.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Icon, Title } from '@/components/atoms';
+import { COLOR } from '@/styles/color';
+import { OptionsType } from '@/components/templates/DetailPostTemplate';
+
+export type OptionsMenuProps = {
+  menuOpenHandler?: () => void;
+  options: OptionsType;
+};
+
+function OptionsMenu({ menuOpenHandler, options }: OptionsMenuProps) {
+  return (
+    <Container>
+      <Overlay>
+        <Wrapper>
+          <Block>
+            <Title style={{ color: COLOR.GRAY07, fontWeight: 'bold' }}>
```

Comment:
인라인 스타일 대신 `styled component` 로 통일 하는것은 어떨까요? 
```jsx
const StyledTitle = styled(Title)`
  font-weight:'bold';
  color : ${({theme} => theme.COLOR07)};
`;

```
```suggestion
            <StyledTitle style={{ color: COLOR.GRAY07, fontWeight: 'bold' }}>
```

Reply:
Title component 누가 만드셨길래 가져다 썼는데 styled-component로 그냥 만들까여 ?

Reply:
이건 기준이 조금 애매하긴 하네요
제가 스타일 컴포넌트를 새로 커스텀하는 기준은

1. 인라인 스타일링이 4개 이상일 때
2. 똑같은 인라인 스타일링이 겹치는 컴포넌트가 있을 때

입니다!